R/standardize.models.R

```diff
@@ -78,6 +78,12 @@ standardize.default <- function(x,
     return(x)
   }
 
+  # check model formula. Some notations don't work when standardizing data
+  if (!insight::formula_ok(x, verbose = verbose)) {
+    insight::format_alert(insight::color_text("Model cannot be standardized.", "red"))
+    return(x)
+  }
```

Comment:
Should we error here? Otherwise it returns wrong results, no?

Reply:
We could error, but otherwise the original input model is returned.

Reply:
IMO it makes more sense to error: if the user wants to standardize the data but it cannot be standardized, it means the function fails so we should force the user to take action.